notifications/views.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render, get_object_or_404
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect
+
+from .models import Notification, clear_old
+
+# Create your views here.
+@login_required
+def all_notifications(request):
+	context = {
+		'notifs': Notification.objects.filter(member=request.user.member).order_by('-unread','-time'),
```

Comment:
Nitpick: maybe it's just me, but shorthand naming seems not useful :P

Reply:
You really, really don't like shorter names do you (fixed)